seed/views/v3/analyses.py

```diff
@@ -72,3 +73,58 @@ def retrieve(self, request, pk):
             'status': 'success',
             'analysis': serialized_analysis
         })
+
+
+class AnalysisPropertyViewViewSet(viewsets.ViewSet):
+    serializer_class = AnalysisPropertyViewSerializer
+    model = AnalysisPropertyView
+
+    @swagger_auto_schema(
+        manual_parameters=[
+            AutoSchemaHelper.query_org_id_field(True),
+            AutoSchemaHelper.query_integer_field('property_id', False, 'Property ID')
+        ]
+    )
+    @require_organization_id_class
+    @api_endpoint_class
+    @ajax_request_class
+    @has_perm_class('requires_member')
+    def list(self, request, analysis_pk):
+        organization_id = int(request.query_params.get('organization_id', 0))
+        try:
+            views_queryset = AnalysisPropertyView.objects.filter(analysis=analysis_pk, analysis__organization_id=organization_id)
+        except AnalysisPropertyView.DoesNotExist:
+            return JsonResponse({
+                'status': 'error',
+                'message': "Requested analysis doesn't exist in this organization."
+            }, status=HTTP_409_CONFLICT)
+        views = []
+        for view in views_queryset:
+            serialized_view = AnalysisPropertyViewSerializer(view).data
+            views.append(serialized_view)
```

Comment:
This definitely seems to work, but DRF has this as well: https://www.django-rest-framework.org/api-guide/serializers/#dealing-with-multiple-objects

Reply:
Ah, right, since I'm not appending something to each view this time I don't need the loop.  Thanks!